.editorconfig

```diff
@@ -16,3 +16,6 @@ indent_style = tab
 
 [md2toc]
 indent_size = 4
+
+[**.java]
+continuation_indent_size = 4
```

Comment:
If indent_size is defined in [*] then continuation_indent_size should be there as well so that it applies to *.xml, Jenkinsfile, *.sql etc.

Reply:
ok. I was trying to be cautious ... Now everywhere